src/OrbitBase/LoggingUtils.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_LOGGING_UTILS_H_
+#define ORBIT_BASE_LOGGING_UTILS_H_
+
+#include <filesystem>
+
+#include "OrbitBase/Result.h"
+#include "absl/strings/str_format.h"
+#include "absl/time/time.h"
```

Comment:
nit: we use <> style includes for everything external to Orbit

Reply:
Changed for all the files I touched. Thanks!